### DISCUSSION TITLE
Responsive design not working

### DISCUSSION THREAD

**Discussion by mrloldev** [tailwindlabs/tailwindcss#12664] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.4
**What build tool (or framework if it abstracts the build tool) are you using?**

postcss-cli 8.3.1, Next.js 14, shadcn

**What version of Node.js are you using?**

For example: v20

**What browser are you using?**

Brave, OperaGX

**What operating system are you using?**

 Windows

**Reproduction URL**
tailwind.config.js ```
/** @type {import('tailwindcss').Config} */
module.exports = {
  darkMode: ["class"],
  content: [
    "./pages/**/*.{ts,tsx}",
    "./components/**/*.{ts,tsx}",
    "./app/**/*.{ts,tsx}",
    "./src/**/*.{ts,tsx}",
  ],
  prefix: "",
  theme: {
    container: {
      center: true,
      padding: "2rem",
      screens: {
        sm: "640px",
        md: "768px",
        lg: "1024px",
        "2xl": "1400px",
      },
    },
    extend: {
      colors: {
        border: "hsl(var(--border))",
        input: "hsl(var(--input))",
        ring: "hsl(var(--ring))",
        background: "hsl(var(--background))",
        foreground: "hsl(var(--foreground))",
        primary: {
          DEFAULT: "hsl(var(--primary))",
          foreground: "hsl(var(--primary-foreground))",
        },
        secondary: {
          DEFAULT: "hsl(var(--secondary))",
          foreground: "hsl(var(--secondary-foreground))",
        },
        destructive: {
          DEFAULT: "hsl(var(--destructive))",
          foreground: "hsl(var(--destructive-foreground))",
        },
        muted: {
          DEFAULT: "hsl(var(--muted))",
          foreground: "hsl(var(--muted-foreground))",
        },
        accent: {
          DEFAULT: "hsl(var(--accent))",
          foreground: "hsl(var(--accent-foreground))",
        },
        popover: {
          DEFAULT: "hsl(var(--popover))",
          foreground: "hsl(var(--popover-foreground))",
        },
        card: {
          DEFAULT: "hsl(var(--card))",
          foreground: "hsl(var(--card-foreground))",
        },
      },
      borderRadius: {
        lg: "var(--radius)",
        md: "calc(var(--radius) - 2px)",
        sm: "calc(var(--radius) - 4px)",
      },
      keyframes: {
        "accordion-down": {
          from: { height: "0" },
          to: { height: "var(--radix-accordion-content-height)" },
        },
        "accordion-up": {
          from: { height: "var(--radix-accordion-content-height)" },
          to: { height: "0" },
        },
      },
      animation: {
        "accordion-down": "accordion-down 0.2s ease-out",
        "accordion-up": "accordion-up 0.2s ease-out",
      },
    },
    screens: {
      sm: "640px",
      md: "768px",
      lg: "1024px",
      "2xl": "1280px",
    },
  },
  plugins: [require("tailwindcss-animate")],
};

```
element ````
      <li className="text-primary/[.8] hover:text-primary transition duration-300   opacity-0  md:opacity-1 ">
              <Link href="/app/exchange">Exchange</Link>
            </li>
```
![image](https://github.com/tailwindlabs/tailwindcss/assets/65485999/422b6828-1dd0-4bfa-9ed4-4271ee11e6b9)

**Describe your issue**

So the md: does not apply aat any case and if I inspect the element that class don't appear and there is no media query. In addition if I hover on the effect at vscode it doesn't show anything, when it does in the opacity-0 
![image](https://github.com/tailwindlabs/tailwindcss/assets/65485999/393fd8fc-6462-4569-96a7-79d683f0aa7e)
![image](https://github.com/tailwindlabs/tailwindcss/assets/65485999/3a91edf2-fbbe-4d51-b5a6-3ed126d4a203)

**Comment by adamwathan** [comment, marked as answer]
Hey! First thing that stands out to me here is that `opacity-1` is not a class in Tailwind, and I don't see any changes to the `opacity` scale in your config. Try `md:opacity-100` instead — as long as your `content` configuration is correct for your project that should solve it.
